### PR TITLE
Remove LF Value Cache

### PR DESCRIPTION
**Pull request by nmarton-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

**Review by tudor-da — APPROVED** [review]
🎆

File: ledger/sandbox-on-x/reference.conf [hunk 1]
```diff
@@ -562,19 +562,6 @@ ledger {
         submission-batch-size = 50
       }
 
-      lf-value-translation-cache {
```
Comment (nmarton-da) [inline ledger/sandbox-on-x/reference.conf:565]: @skisel I assume we need to delete this. Is there something else to do if we are removing configurations?

File: ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala [hunk 2]
```diff
@@ -576,16 +571,9 @@ object CliConfig {
...
-        )
-        .action((maximumLfValueTranslationCacheEntries, config) =>
-          config.copy(
-            lfValueTranslationEventCache = config.lfValueTranslationEventCache
-              .copy(maximumSize = maximumLfValueTranslationCacheEntries),
-            lfValueTranslationContractCache = config.lfValueTranslationContractCache
-              .copy(maximumSize = maximumLfValueTranslationCacheEntries),
-          )
-        ),
+          s"Deprecated! This configuration parameter has no effect. The maximum size of the cache used to deserialize Daml-LF values, in number of allowed entries. By default, nothing is cached."
```
Comment (nmarton-da) [inline ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala:574]: @mziolekda Is it okay to communicate deprecation like this? Do we need to make this maybe hidden a well?

File: ledger/sandbox-on-x/reference.conf (hunk 1 above)
Comment (skisel-da) [reply ledger/sandbox-on-x/reference.conf:565]: I don't see any other usage, so it's okay

Comment (skisel-da) [reply ledger/sandbox-on-x/reference.conf:565]: btw, you've tagged a wrong skisel ;)

File: ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala (hunk 2 above)
Comment (mziolekda) [inline ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala:574]:
```suggestion
          s"Deprecated parameter --  lf value translation cache doesn't exist anymore."
```

Comment (mziolekda) [reply ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala:574]: That's how we've been doing it in oem kit

**Review by mziolekda — APPROVED** [review]

**Pull request merged**